docs/operating-scylla/admin-tools/scylla-sstable.rst

```diff
@@ -99,40 +96,529 @@ Dropped columns can be provided to the tool in the form of insert statements int
+.. code-block:: none
+    :class: hide-copy-button
+
+    $ROOT := { "$sstable_path": Bool, ... }
+
+decompress
+^^^^^^^^^^
+
+Decompress Data.db if compressed. Noop if not compressed. The decompressed data is written to Data.db.decompressed.
+E.g. for an sstable:
+
+.. code-block:: console
+    :class: hide-copy-button
+
+    md-12311-big-Data.db
+
+the output will be:
+
+.. code-block:: console
+    :class: hide-copy-button
+
+    md-12311-big-Data.db.decompressed
+
+write
+^^^^^
+
+Write an sstable based on a JSON representation of the content.
+The JSON representation has to have the same schema as that of a single sstable from the output of the `dump-data operation <dump-data_>`_ (corresponding to the ``$SSTABLE`` symbol).
+The easiest way to get started with writing your own sstable is to dump an existing sstable, modify the json then invoke this operation with the result.
+You can feed the output of dump-data to write by filtering the output of the former with ``jq .sstables[]``:
+
+.. code-block:: console
+
+    scylla sstable dump-data --system-schema system_schema.columns /path/to/me-14-big-Data.db | jq .sstables[] > input.json
+    scylla sstable write --system-schema system_schema.columns --input-file ./input.json --generation 0
+    scylla sstable dump-data --system-schema system_schema.columns ./me-0-big-Data.db | jq .sstables[] > dump.json
+
+At the end of the above, ``input.json`` and ``dump.json`` will have the same content.
+
+Note that `write` doesn't yet support all the features of the scylladb storage engine. The following is not supported:
+
+* Counters.
+* Non-strictly atomic cells, this includes frozen multi-cell types like collections, tuples and UDTs.
+
+Parsing uses a streaming json parser, it is safe to pass in input-files of any size.
+
+The output sstable will use the BIG format, the highest supported sstable format and the specified generation (``--generation``).
+By default it is placed in the local directory, can be changed with ``--output-dir``.
+If the output sstable clashes with an existing sstable, the write will fail.
+
+The output is validated before being written to the disk.
+The validation done here is similar to that done by the `validate operation <validate_>`_.
+The level of validation can be changed with the ``--validation-level`` flag.
+Possible validation-levels are:
+
+* ``partition_region`` - only check fragment types, e.g. that a partition-end is followed by partition-start or EOS;
+* ``token`` - also check token order of partitions;
+* ``partition_key`` - full check on partition-ordering;
+* ``clustering_key`` - also check clustering element ordering;
+
+Note that levels are cumulative, each contains all the checks of the previous levels too.
+By default the strictest level is used.
+This can be relaxed if e.g. one wants to produce intentionally corrupt sstables for tests.
```

Comment:
```suggestion
Note that levels are cumulative - each contains all the checks of the previous levels, too.
By default, the strictest level is used.
This can be relaxed, for example, if you want to produce intentionally corrupt SStables for tests.
```